spot_wrapper/wrapper.py

```diff
@@ -117,6 +117,42 @@
     "ImageBundle", ["frontleft", "frontright", "left", "right", "back", "hand"]
 )
 
+# Added to support getting images for given cameras
+IMAGE_SOURCES_BY_CAMERA = {
```

Comment:
All of these should be an enum rather than strings.

Reply:
While typing the code below, I think.... typo can still happen.. (unless, we do static checking of course).
```python
IMAGE_SOURCES_BY_CAMERA = {
    Camera.frontleft: {
        ImageType.visual: ImageSource.frontleft_fisheye_image,
        ImageType.depth: ImageSource.frontleft_depth,
        ImageType.depth_registered: frontleft_depth_in_visual_frame,
    },
   ...
```

Reply:
In any case, I did this. See the latest commit

Reply:
I don't think we should use an enum here. The underlying spot API provides strings, and is actually somewhat flexible with them. For example, you can define image sources for extensions that you attach to spot. If we do that, then we'd need to add a new enum here.

Reply:
I would spend the time to extend the spot_wrapper to provide the list of camera sources, and use that as the list. If you don't want to spend that time, I would open a jira ticket, and hard-code a list of camera sources at the top, with a comment linking to that jira ticket.

Reply:
I was previously hard-coding the strings to be consistent with the other code in spot wrapper (because there's already a lot of hard-coded source names).  
I would be fine with leaving hard-coded strings at the top of spot wrapper because those strings would be written only once.   

That said, I'll remove the Enum for ImageSource and revert back to the strings. Those strings shouldn't need to be typed by the user of SpotWrapper anyways. I could open a JIRA ticket in the PickNik board about this.

I think the dataclass `CameraSource` is worthwhile to add, which clarifies the input type to `get_images_by_cameras`.

Reply:
Sounds good to me!

Reply:
Ticket created: https://theaiinstitute.atlassian.net/browse/PN-5

I will mark this as resolved.  I reverted adding the enums, and tested again on hardware.  The functionality works.